src/include/daos_security.h

```diff
@@ -138,6 +153,32 @@ enum daos_acl_perm {
 	DAOS_ACL_PERM_SET_OWNER		= (1U << 8),
 };
 
+/**
+ * Mask of all valid permissions for DAOS pools
+ */
+#define DAOS_ACL_PERM_POOL_ALL	(DAOS_ACL_PERM_READ |			\
+				 DAOS_ACL_PERM_WRITE |			\
+				 DAOS_ACL_PERM_CREATE_CONT |		\
+				 DAOS_ACL_PERM_DEL_CONT)
+
+/**
+ * Mask of all valid permissions for DAOS containers
+ */
+#define DAOS_ACL_PERM_CONT_ALL	(DAOS_ACL_PERM_READ |			\
+				 DAOS_ACL_PERM_WRITE |			\
+				 DAOS_ACL_PERM_DEL_CONT |		\
```

Comment:
It does indeed. This way someone could be granted permission to delete only a specific container, rather than being allowed to delete all the containers in the pool.